### ISSUE TITLE
Is the magnussolutions.com Telegram still the valid contact method?

### ISSUE THREAD

**Issue by crisicon** [magnussolution/magnusbilling#599]
Hi there,
I paid for one-hour support and tried to contact you via the Telegram account posted on the website.
Did not receive back any reply...

Are you still active?

**Comment by xceedconnections** [comment]
YES,  that's the only channel you can contact the developer for the paid support.

**Comment by crisicon** [comment]
Well... I left a message about the paid hour about a week ago and nobody has replied, nor contacted me, including after I have sent an email to the address listed on the website...
Hopefully someone will, in the near future, otherwise I'll deem it an unsupported solution and move away from it.
It would be a pity as I was just about to seek a more permanent, long term relationship.

**Comment by rotechy** [comment]
hello there, tell me about your issue. let's look at it.

**Comment by crisicon** [comment]
Hi @rotechy,
I'm trying to change the configuration so the calls are no longer limited to 60 seconds.
I managed to do this for outbound calls, by adding L(14400000) to each trunk, so they are now allowed calls 4 hours long.

I can't figure out why the limit is still there for inbound.

Thanks for offering your help
Cris
